app/src/main/java/org/mozilla/focus/ext/EngineView.kt

```diff
@@ -109,6 +110,16 @@ fun EngineView.scrollByClamped(vx: Int, vy: Int) {
 val EngineView.focusedDOMElement: FocusedDOMElementCache
     get() = getOrPutExtension(this).domElementCache
 
+fun EngineView.saveState(): Bundle {
+    val bundle = Bundle()
+    getOrPutExtension(this).webView.saveState(bundle)
```

Comment:
Looks like from the docs that this saves the back/forward state, but not the display state.

Reply:
Yeah, I was hesitant about that too.  We had a big talk about it on slack (https://mozilla.slack.com/archives/C8P1Z6JSZ/p1539383487000100).  I'll add a comment about it here.